__tests__/setup-go.test.ts

```diff
@@ -230,6 +231,22 @@ describe('setup-go', () => {
     expect(logSpy).toHaveBeenCalledWith(`Setup go version spec 1.13.0`);
   });
 
+  it('exports GOROOT', async () => {
```

Comment:
```suggestion
  it('does not export any variables', async () => {
```

Looks like the test is making sure we *don't* export `GOROOT`, correct?